apisix/plugins/jwt-auth.lua

```diff
@@ -435,8 +412,7 @@ function _M.rewrite(conf, ctx)
         return 401, {message = "Missing related consumer"}
     end
 
-    local consumers = lrucache("consumers_key", consumer_conf.conf_version,
-        create_consume_cache, consumer_conf)
+    local consumers = consumer_mod.consumers_kv(plugin_name, consumer_conf, "key")
```

Comment:
Should be "user_key"?

Reply:
No, look: https://github.com/apache/apisix/blob/master/apisix/plugins/jwt-auth.lua#L149
```
consumer_names[consumer.auth_conf.key] = consumer
```